cmd/main.go

```diff
@@ -83,6 +87,7 @@ func main() {
 	flag.DurationVar(&gardenerRequestTimeout, "gardener-request-timeout", defaultGardenerRequestTimeout, "Timeout duration for requests to Gardener")
 	flag.BoolVar(&enableRuntimeReconciler, "runtime-reconciler-enabled", defaultRuntimeReconcilerEnabled, "Feature flag for all runtime reconciler functionalities")
 	flag.BoolVar(&persistShoot, "persist-shoot", false, "Feature flag to allow persisting created shoots")
+	flag.StringVar(&converterConfigFilepath, "converter-config-filepath", "converter_config.json", "A file path to the gardener shoot coverter configuration.")
```

Comment:
The `converter_config.json` is not present in this PR. Should it be `converter_config.yaml` instead?

Reply:
Thx,
- I will add the configuration to hack dir
- Fix the typo in the description